src/Components/Products/Products.jsx

```diff
@@ -0,0 +1,42 @@
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const Products = ({ products }) => (
+  <div className="mx-5">
+    <div className="grid grid-cols-2">
+      {products && products.map((product) => (
+        <div key={product.id} className="basis-1/4 flex justify-center flex-col text-center items-center p-2 cursor-pointer hover:border border-gray-700">
+          <NavLink to={`/product/${product.id}`} state={product}>
+            <img
+              className="object-contain w-[200px] aspect-square object-contain"
+              src={product.image}
+              alt="picture1"
+            />
+            <div className="mt-4 flex flex-col items-center">
+              <p className="font-bold">{product.category}</p>
+              <p>{product.title}</p>
+              <p className="font-extrabold">
+                $
+                {' '}
+                {product.price}
+              </p>
+            </div>
+          </NavLink>
+        </div>
+      )) }
+    </div>
+  </div>
+);
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
  -  Typography: font face, size, and weight
  -  Layout: composition and space between elements.
**Here is how look the home pages :point_down:**

| Template design | Your Design |
| --------------------- | ------------------- |
| ![Screenshot from 2023-01-12 23-41-09](https://user-images.githubusercontent.com/83042742/212187372-361a924d-2991-4605-b23e-11536cc82caa.png) | ![Screenshot from 2023-01-12 23-40-09](https://user-images.githubusercontent.com/83042742/212187437-e84142fc-abe0-4556-9ca2-b9829484182b.png) |